### PR TITLE
update what's new

### PR DESCRIPTION
**Pull request by Songyuanwei**
Thank you for your contribution to the MindOCR repo.
Before submitting this PR, please make sure:

- [x] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [ ] You have added unit tests

## Motivation

(Write your motivation for proposed changes here.)

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: README_CN.md [hunk 1]
```diff
@@ -155,14 +155,46 @@ MindSpore Lite和ACL模型推理的支持列表，请见[MindOCR模型推理支
...
+1. 支持加载自定义的预训练checkpoint， 通过在yaml配置中将`model-pretrained`设置为checkpoint url或本地路径来使用。
+2. 支持设置执行包括旋转和翻转在内的数据增强操作的概率。
+3. 为模型训练添加EMA功能，可以通过在yaml配置中设置`train-ema`（默认值：False）和`train-ema_decay`来启用。
+4. 参数修改：`num_columns_to_net` -> `net_input_column_index`: 输入网络的columns数量改为输入网络的columns索引
+5. 参数修改：`num_columns_of_labels` -> `label_column_index`: 代表label的columns数量改为代表label的columns索引
+
+- 2023/04/21
+1. 添加参数分组以支持训练中的灵活正则化。用法：在yaml config中添加`grouping_strategy`参数以选择预定义的分组策略，或使用`no_weight_decay_params`参数选择要从权重衰减中排除的层（例如，bias、norm）。示例可参考`configs/rec/crn/crnn_icdar15.yaml`
+2. 添加梯度积累，支持大批量训练。用法：在yaml配置中添加`gradient_accumulation_steps`，全局批量大小=batch_size * devices * gradient_aaccumulation_steps。示例可参考`configs/rec/crn/crnn_icdar15.yaml`
+3. 添加梯度裁剪，支持训练稳定。通过在yaml配置中将`grad_clip`设置为True来启用它。
```
Comment (SamitHuang) [inline README_CN.md:193]: 部分机翻痕迹过重，要调整下：如启用它-> 启用，

5. 参数修改：`num_columns_of_labels` -> `label_column_index`: 代表label的columns数量改为代表label的columns索引  看不懂啥意思

File: README_CN.md [hunk 2]
```diff
@@ -155,14 +155,46 @@ MindSpore Lite和ACL模型推理的支持列表，请见[MindOCR模型推理支
...
+3. 增加断点重训(resume training)功能，可在训练意外中断时使用。如需使用，请在配置文件中`model`字段下增加`resume`参数，允许传入具体路径`resume: /path/to/train_resume.ckpt`或者通过设置`resume: True`来加载在ckpt_save_dir下保存的trian_resume.ckpt
+
+
+- 2023/05/15
+1. 增加新模型
+    - 文本检测[DBNet++](configs/det/dbnet)
+    - 文本识别[CRNN-Seq2Seq](configs/rec/rare)
+    - 在SynthText数据集上预训练的[DBNet](https://download.mindspore.cn/toolkits/mindocr/dbnet/dbnet_resnet50_synthtext-40655acb.ckpt)
+2. 添加更多基准数据集及其结果
+    - [SynthText](https://academictorrents.com/details/2dba9518166cbd141534cbf381aa3e99a087e83c), [MSRA-TD500](docs/cn/datasets/td500_CN.md), [CTW1500](docs/cn/datasets/ctw1500_CN.md)
```
Comment (HaoyangLee) [inline README_CN.md:177]: SynthText链接无效。建议改为https://www.robots.ox.ac.uk/~vgg/data/scenetext/

File: README.md [hunk 3]
```diff
@@ -164,6 +164,18 @@ After downloading these datasets in the `DATASETS_DIR` folder, you can run `bash
...
+- 2023/06/07
+1. Add new trained models
+    - [PSENet](configs/det/psenet) for text detection
+    - [EAST](configs/det/east) for text detection
+    - [SVTR](configs/rec/svtr) for text recognition
+2. Add more benchmark datasets and their results 
+    - [totaltext](docs/en/datasets/totaltext.md)
+    - [mlt2017](docs/en/datasets/mlt2017.md)
+    - [chinese_text_recognition](docs/en/datasets/chinese_text_recognition.md)
+3. Add resume training function, which can be used in case of unexpected interruption in training. Usage: add the `resume` parameter under the `model` field in the yaml config. Allow specific paths to be passed in `resume: /path/to/train_resume.ckpt` or load it in ckpt_save_dir by setting `resume: True`.
```
[the remark this answers is not in the document]
Comment (Songyuanwei) [reply README.md]: fixed

File: README_CN.md (hunk 1 above)
Comment (Songyuanwei) [reply README_CN.md]: fixed

File: README_CN.md (hunk 2 above)
Comment (Songyuanwei) [reply README_CN.md]: 我可以打开这个链接，不过还是修改了链接地址

Comment (HaoyangLee) [reply README_CN.md]: 更正：该链接中显示The download links for the SynthText dataset are no longer available from this website. 无法下载数据集。
建议用回原链接https://academictorrents.com/details/2dba9518166cbd141534cbf381aa3e99a087e83c

> SynthText链接无效。建议改为https://www.robots.ox.ac.uk/~vgg/data/scenetext/

**Review by HaoyangLee — APPROVED** [review]

**Review by SamitHuang — APPROVED** [review]

**Pull request merged**